src/cli.coffee

```diff
@@ -35,6 +35,7 @@ watchCmd = program
   .option('-p, --port [port]', 'if a `server` option was specified, define on which port
  the server would run')
   .option('-d, --debug', 'print verbose debug output to stdout')
+  .option('-l, --stdin', 'listen to stdin and shut down when stdin closes')
```

Comment:
It's your PR :smile:

I don't have strong feelings about it, but tempted to say I'd prefer leaving it at just the explicit `--stdin`. It's not going to be manually typed in enough (or ever) to justify a shorthand letter.